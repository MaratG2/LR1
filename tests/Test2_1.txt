-37528630681841519304805018382188409536.525712
225157823139443265843969155167523849608.917771
119618976754110859743452230939977945902.102900