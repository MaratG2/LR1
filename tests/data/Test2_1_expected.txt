-37528631225178701761487703823739781120,00000000000000000000
